packages/starlight/user-components/Tabs.astro

```diff
@@ -77,7 +77,7 @@ const { html, panels } = processPanels(panelHtml);
 			super();
 			const tablist = this.querySelector<HTMLUListElement>('[role="tablist"]')!;
 			this.tabs = [...tablist.querySelectorAll<HTMLAnchorElement>('[role="tab"]')];
```

Comment:
Don’t we need a similar approach for this selector too?

Reply:
We actually do not (which is quite neat as those are not direct children and would need longer selectors):

- The first one is using `querySelector()` and not `querySelectorAll()` so it only return the first result which is what we want.
- The second one is re-using the result of the first one so it's also what we want.

Basically only the last one was an issue as it was based on the component (`this`) and was using `querySelectorAll()`.